hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/streaming/HoodieSourceOffset.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.sql.execution.streaming.{Offset, SerializedOffset}
 
 case class HoodieSourceOffset(commitTime: String) extends Offset {
 
-  override def json(): String = {
+  override val json: String = {
```

Comment:
also the change is necessary? would you please clarify?

Reply:
This just is an improve change, json no need to recalculate when calling this again, as it's value can not be changed.